src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceSpecifications.kt

```diff
@@ -57,11 +57,7 @@ fun hasNomsIdIn(nomsIds: List<String>?): Specification<Licence>? = nomsIds?.let
 
 fun hasResponsibleComIn(staffIds: List<Int>?): Specification<Licence>? = staffIds?.let {
   return Specification<Licence> { root, _, builder ->
-    if (staffIds.isNullOrEmpty()) {
```

Comment:
Ah, because it wouldn't go in here in the first place if there was nothing in the list